### ISSUE TITLE
Internal server error not user-friendly at all

### ISSUE THREAD

**Issue by carueda** [RDFLib/pyLODE#88]
Simply put, pyLODE should handle invalid input gracefully.

http://cor.esipfed.org/pylode?url=INVALID :

> ![image](https://user-images.githubusercontent.com/556505/90288573-ff38dd00-de2e-11ea-9504-6073efb26ce7.png)

Along with any internal logging or stack traces, the exposed output should indicate what went wrong at least in general terms for the more casual visitor, for example, indicating the the URI could not be resolved, or that it was not resolved to an unrecognized format, etc.

**Comment by nicholascar** [comment]
Yes, it should. I will try and improve internal error surfacing.

**Comment by ananya2711** [comment]
Does the service on http://cor.esipfed.org/pylode still run? Doesn't load for me.

**Comment by nicholascar** [comment]
Sorry but that is a question you'll have to put to ESIP! 

Since they implemented that live-loading version, there has been a major update to pyLODE (v3.x) that includes many breaking changes. If their system auto-pulls pyLODE from PyPI and tries to run with it, it will have broken.

I think this new version of pyLODE is better (duh!) but it will need to be catered for in some changes at their end.
